pytest/tests/mocknet/run_adversenet.py

```diff
@@ -197,5 +226,8 @@ def get_role(node):
             all_nodes, chain_id, override_config)
     else:
         mocknet.update_existing_config_files(all_nodes, override_config)
-    mocknet.start_nodes(all_nodes)
+    env = {}
```

Comment:
nit: consider adding flags `RUST_BACKTRACE=full RUST_LOG=debug,actix_web=info` from `neard_start_script` as well.
I'd like to suggest to use `env` variable in `Machine.run(...)` as well, but it requires the change in the python-rc package.